frontend/src/citizen-frontend/messages/MessagesPage.tsx

```diff
@@ -46,17 +44,17 @@ export default React.memo(function MessagesPage() {
 
         return state
       }),
-    [setBulletinsState]
+    [setMessagesState]
```

Comment:
`setMessagesState` is not needed in the dependency array, because `useState` callbacks are stable.